recipes/mpiFFT4py/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "mpiFFT4py" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/spectralDNS/{{ name }}.git
+  git_tag: {{ version }}
```

Comment:
Can we please use a tarball instead of cloning from `git`? Normally GitHub provides `.tar.gz` that we can use. Also please include a checksum. `sha256` is preferred.